src/core/Entry.cpp

```diff
@@ -1059,6 +1059,15 @@ QString Entry::resolveMultiplePlaceholdersRecursive(const QString& str, int maxD
         return str;
     }
 
+    // Short circuit if we have escaped the placeholder brackets
+    if (str.startsWith("\\{") && str.endsWith("\\}")) {
+        // Replace the escaped brackets with actuals and move on
+        auto ret = str;
+        ret.replace(0, 2, "{");
+        ret.replace(ret.size() - 2, 2, "}");
+        return ret;
+    }
+
```

Comment:
This is a very brittle escaping mechanism. Either we need a proper state machine for parsing the syntax or this should at least be part of the regex, e.g. as negative lookbehinds/lookaheads.

Reply:
@phoerious 
This is a very special case situation and should rarely be encountered. Since we properly test the start and end conditions before manually manipulating the string, I don't see a particular problem with this implementation. I am not interested in putting extensive work to make a state machine or complex regex just for this.